File: TutorialGameVB/TutorialGameVB/Game/TutorialGame.vb
Imports System
Imports SFML.Audio
Imports SFML.Graphics
Imports SFML.Window
Imports SFML.System

Namespace TutorialGame

    Public Class TutorialGame
        Inherits GameLoop

        Public Const WINDOW_WIDTH As UInteger = 640

        Public Const WINDOW_HEIGHT As UInteger = 480

        Public Const WINDOW_TITLE As String = "Tutorial Game"

        Public Shared WINDOW_COLOR As Color = Color.Black

        Public Shared WINDOW_STYLE As Styles = Styles.Default

        Public Sub New()
            MyBase.New(WINDOW_WIDTH, WINDOW_HEIGHT, WINDOW_TITLE, WINDOW_COLOR, WINDOW_STYLE)
        End Sub

        Public Overrides Sub LoadContent()
            DebugUtility.LoadContent()
        End Sub

        Public Overrides Sub Initialize()
        End Sub

        Public Overrides Sub Update(ByVal gameTime As GameTime)
        End Sub

        Public Overrides Sub Draw(ByVal gameTime As GameTime)
            DebugUtility.DrawPerformanceData(Me, Color.White)
        End Sub
    End Class

End Namespace

File: TutorialGame/TutorialGame/Game/TutorialGame.cs
using System;
using SFML.Audio;
using SFML.Graphics;
using SFML.Window;
using SFML.System;

namespace TutorialGame
{
    public class TutorialGame : GameLoop
    {
        public const uint WINDOW_WIDTH = 640;
        public const uint WINDOW_HEIGHT = 480;
        public const string WINDOW_TITLE = "Tutorial Game";
        public static Color WINDOW_COLOR = Color.Black;
        public static Styles WINDOW_STYLE = Styles.Default;

        public TutorialGame() : base (WINDOW_WIDTH, WINDOW_HEIGHT, WINDOW_TITLE, WINDOW_COLOR, WINDOW_STYLE)
        {
            
        }

        public override void LoadContent()
        {
            DebugUtility.LoadContent();
        }

        public override void Initialize()
        {
        }

        public override void Update(GameTime gameTime)
        {
        }

        public override void Draw(GameTime gameTime)
        {
            DebugUtility.DrawPerformanceData(this, Color.White);
        }
    }
}
